gspread/models.py

```diff
@@ -813,6 +813,96 @@ def findall(self, query):
         """
         return list(self._finder(filter, query))
 
+    def freeze(self, rows=None, cols=None):
```

Comment:
You probably want to include some `@property` methods `frozen_row_count` and `frozen_column_count` so that these values are readable from the Worksheet object.

Reply:
Fair point. The only issue is that this will be affected by #545 (see #548). Added them anyway for now.